entproto/extension.go

```diff
@@ -101,11 +112,16 @@ func Hook() gen.Hook {
 	return x.hook()
 }
 
-// Generate takes a *gen.Graph and creates .proto files. Next to each .proto file, Generate creates a generate.go
+// Generate takes a *gen.Graph and creates .proto files.
+// Next to each .proto file, Generate creates a generate.go
 // file containing a //go:generate directive to invoke protoc and compile Go code from the protobuf definitions.
 // If generate.go already exists next to the .proto file, this step is skipped.
+// You can use entproto.WithProtoDir to set the directory where the generated .proto files will be written.
+// You can use entproto.SkipGenFile to disable the generation of the generate.go file.
 func Generate(g *gen.Graph) error {
-	x := &Extension{}
+	x := &Extension{
+		skipGenFile: true,
```

Comment:
Skip should be false by default (to preserve the current behavior)

Reply:
sorry, i forget to remove it